qiita_pet/templates/private_studies.html

```diff
@@ -134,6 +137,22 @@ <h4 class="modal-title" id="myModalLabel">Modify Sharing Settings</h4>
             {% end %}
         </tbody>
     </table>
+
+<!--User study Modals-->
+{% for s in user_studies %}
```

Comment:
A way to do this would be to:
1.- add a single div with the modal window.
2.- dynamically populate this modal window using the information that
already exists in the data table.
3.- When the button "show abstract" button is clicked, populate the
single div in window 1 (this should be easily done with jQuery).

If this is still not clear, we can chat off-line.

On (Mar-12-15|10:53), Joshua Shorenstein wrote:
> > @@ -134,6 +137,22 @@ <h4 class="modal-title" id="myModalLabel">Modify Sharing Settings</h4>
> >              {% end %}
> >          </tbody>
> >      </table>
> > +
> > +<!--User study Modals-->
> > +{% for s in user_studies %}
> 
> Yes. How do you propose doing this in javascript? I'm not sure I can even think of a way to do so. This is the easiest way to do it, sonce it makes the modals and shows them as needed.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/970/files#r26326871